languages/gutenberg.pot

```diff
@@ -33,16 +33,22 @@ msgid "List"
 msgstr ""
 
 #: blocks/library/list/index.js:25
+#: blocks/library/quote/index.js:38
+#: blocks/library/quote/index.js:37
```

Comment:
Appears these got doubled up. Another `npm run build` should clear it up. Not certain what causes this.